.github/workflows/_charm-codeql-analysis.yml

```diff
@@ -4,7 +4,7 @@ name: "CodeQL"
 on:
   workflow_call:
     inputs:
-      charm-path:
+      resource-path:
```

Comment:
`resource` may be confusing because it's a term we already have for charms (charm resource).
If we were to change it then perhaps use [charmhub's term - "package"](https://api.charmhub.io/docs/default.html#package_metadata).
But not sure it's worth the effort.